pages/api/journeysFR.json

```diff
@@ -0,0 +1,564 @@
+            },
+            {
+              "title": "Fournisseurs de services communs :",
+              "list": [
+                "Professionnels de la santé",
+                "Médecins spécialistes",
+                "Médecin de famille",
+                "Birthing professional",
+                "Hospital/Clinic/Birthing centre **",
+                "Travailleur social *",
+                "Spécialiste de l’adoption *",
+                "Immigration. Réfugiés et Citoyenneté Canada *",
+                "Unité de soins intensifs nouveau-né"
+              ]
+            },
+            {
+              "title": "",
+              "list": [
+                "* Specific to the adoption path french",
+                "** Specific to giving birth french"
+              ]
+            }
+          ]
+        },
+        {
+          "title": "Inscription de l’enfant",
+          "content": [
+            {
+              "title": "Étapes Commun :",
+              "list": ["présenter une demande de certificat de naissance"]
+            },
+            {
+              "title": "Fournisseurs de services communs :",
+              "list": [
+                "Hôpital/Clinique/Centre de naissances",
+                "Gouvernement provincial"
+              ]
+            }
+          ]
+        },
+        {
+          "title": "Le cheminement du deuil s'amorce",
+          "content": [
+            {
+              "title": "Common situations:",
+              "list": [
+                "Décès d’un nourrisson",
+                "Accouchement d'un mort-né",
+                "Décès d’un parent au moment de l’accouchement"
+              ]
+            },
+            {
+              "title": "Étapes Commun :",
+              "list": [
+                "Analyse et rapports médicaux suivant la perte du bébé",
+                "Préparer les arrangements de fin de vie",
+                "Soutien en matière de santé mentale et émotionnelle",
+                "Garder leur mémoire vivante"
+              ]
+            },
+            {
+              "title": "Fournisseurs de services communs :",
+              "list": [
+                "Professionnels de la santé",
+                "Médecin de famille",
+                "Professionnels de l’accouchement",
+                "Hôpital/Clinique/Centre des naissances",
+                "Professionnels de la santé mentale",
+                "Groupes de soutien",
+                "Organismes de bienfaisance/Organismes à but non-lucratif",
+                "Généticien",
+                "Spécialistes de la médecine fœtale",
+                "Médecins spécialistes",
+                "Analyse médicale",
+                "Médecins chargés des grossesses à haut risque",
+                "Travailleur social",
+                "Professionnels de la santé et du bien être"
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "title": "transition vers la nouvelle normalité",
```

Comment:
First letter should be capitalized here

Reply:
agree but this is not the final french translations